src/Controller/CookieAdminListController.php

```diff
@@ -20,13 +21,21 @@ class CookieAdminListController extends AdminListController
     /** @var AdminListConfiguratorInterface */
     private $configurator;
 
+    /** @var DomainConfiguration */
+    private $domainConfiguration;
+
+    public function __construct(DomainConfiguration $domainConfiguration)
```

Comment:
Should it be better to typehint against the `Kunstmaan\AdminBundle\Helper\DomainConfigurationInterface` interface here?